Add new Advanced Setting for Throughput Limits (#606)
mongodb/docs-app-services: source/triggers/database-triggers.txt

     - .. include:: /includes/trigger-auto-resume.rst

   * - :guilabel:`Maximum Throughput Triggers`

     - .. _triggers-maximum-throughput:
      

>        If your App Services app is running on a dedicated server (M10+ Tier), 

Reviewer: ```suggestion
       If the linked data source is a dedicated server (M10+ Tier),
```

>        you can increase the maximum throughput from the default 10K to 50K. 

Reviewer: Can we link to the [Event Processing Throughput](https://www.mongodb.com/docs/atlas/app-services/triggers/#event-processing-throughput) limitation?

Also we don't define throughput on this page - can we clarify that 10K and 50K refer to the maximum number of events that can be processed simultaneously

     - .. include:: /includes/trigger-auto-resume.rst

   * - :guilabel:`Maximum Throughput Triggers`

     - .. _triggers-maximum-throughput:
      
       If the linked data source is a dedicated server (M10+ Tier), 
       you can increase the :ref:`maximum throughput <event_processing_throughput>`
       from the default 10,000 concurrent processes to 50,000. 

>        Before increasing the maximum throughput, consider whether one or more of 
>        your triggers are calling a rate-limited external API. Increasing the 

Reviewer: ```suggestion
       Before increasing the maximum throughput, consider whether the
       trigger is calling a rate-limited external API. Increasing the 
```

The limit & setting are per-trigger so you really only need to consider the individual triggers you enable
